packages/cli/src/command-helpers/file-resolver.ts

```diff
@@ -39,7 +39,7 @@ export async function resolveFile(
     try {
       // If it's an IPFS hash (Qm...)
       if (source.startsWith('Qm')) {
-        const response = await fetch(`${DEFAULT_IPFS_URL}/cat?arg=${source}`);
+        const response = await fetch(`${getGraphIpfsUrl().ipfsUrl}/cat?arg=${source}`);
```

Comment:
This is wrong. The `/api/v0/cat` endpoint is a POST endpoint and only works with GET atm. due to backwards compatibility. All GET requests should go to `/ipfs/{cid}`.